specs/integrations/synthetics.md

```diff
@@ -0,0 +1,42 @@
+## Synthetics Integration
+
+Synthetic monitors play a crucial role in periodically checking the status of your services and applications on a global scale. General documentation about synthetic monitors can be found in
+[Synthetics getting started page](https://www.elastic.co/guide/en/observability/current/synthetics-get-started.html).
+
+This integration goes in to more detail about how the sythetics monitors would
+be correlated with the APM traces. Synthetics traces can be categorized in to two
+main types
+  1. HTTP checks - These have one-one mapping with APM transactions
+  2. Browser checks - These have a one-to-many mapping with APM transactions
+
+### Correlation
+
+The Synthetics agent takes the responsibility of creating the [`traceparent`](../agents/tracing-distributed-tracing.md#trace_id-parent_id-and-traceparent) header for each outgoing network request associated with a test during every monitor execution.
+
+- `trace.id` and `parent.id`
+  - outgoing requests that are being explicity traced by the synthetics agent
+     will have the `parent.id` and `trace.id` as part of the trace context.
+  - must be unique for each step for a browser monitor
+  - must be unique for a http monitor
+- `sampled` Flag
+  - used to control the sampling decision for all the downstream services.
+  - 100% sampling when tracing is enabled
+
+These correlation values would be applicable even if the downstream services are traced by
+OpenTelemetry(OTEL)-based agents.
+
+### Identifying Synthetics trace
+
+Synthetics monitor executions creates `rootless traces` as these traces are not
+reported to the APM server. To overcome this limitation on the APM UI, we need
+to identify the synthetics traces and explicity link them  to the Synthetics
+waterfall view. 
+
+- `http.headers.user-agent`:
+  - Contains `Elastic/Synthetics` for all outgoing requests from Synthetis based monitors.
```

Comment:
What are the limitations of that? For example, what happens when users have changed the user-agent header? Not saying we absolutely need to cater for that scenario, especially in the first iteration, but it would be useful to document the limitations of that approach.

Reply:
Good point, I will also add the future use case for tracestate if we ever need that scenario.

Reply:
Done - https://github.com/elastic/apm/pull/825/commits/dadb42b77b23b117dfb575a36723f50d711c6d5e